src/store/RelayRecordStore.js

```diff
@@ -129,6 +131,7 @@ class RelayRecordStore {
     if (identifyingArgValue == null) {
       identifyingArgValue = EMPTY;
     }
+    identifyingArgValue=stableStringify(identifyingArgValue);
```

Comment:
nit: please try to stay consistent with our code formatting. we prefer whitespace around assignments: `var = fn(value)`